.github/workflows/test.yml

```diff
@@ -26,10 +26,32 @@ jobs:
         pip install tox tox-gh-actions
 
     - name: Run tox
-      run: tox
+      run: tox  # since python-version is set above it calls py3-etc. not docs
```

Comment:
I believe the reason launching tox with this command does not run docs is because the gh-actions section in tox.ini only points to py3, and not docs

Reply:
I updated the comment

Reply:
looks good, thanks